README.md

```diff
@@ -91,6 +91,8 @@
 * [`toSnakeCase`](#tosnakecase)
 * [`truncateString`](#truncatestring)
 * [`words`](#words)
+* [`stringToIntegers`](#stringToIntegers)
```

Comment:
The hash fragment should be all lower case otherwise links will not work.

Reply:
Thanks for the input. Was struggling with it.